packages/stryker/src/input/InputFileCollection.ts

```diff
@@ -17,14 +17,14 @@ export default class InputFileCollection {
         No files selected. Please make sure you either run stryker a git repository context (and don't specify \`files\` in your stryker.conf.js file),
         or specify the \`files\` property in your stryker config.`));
     } else {
-      if (this.mutateFileNames.length) {
-        log.info(`Found ${this.mutateFileNames.length} of ${this.files.length} file(s) to be mutated.`);
+      if (this.filesToMutate.length) {
+        log.info(`Found ${this.filesToMutate.length} of ${this.files.length} file(s) to be mutated.`);
       } else {
         log.warn(`No files marked to be mutated, stryker will perform a dry-run without actually mutating anything.`);
       }
       if (log.isDebugEnabled) {
         log.debug(`All input files: ${JSON.stringify(this.files.map(file => file.name), null, 2)}`);
-        log.debug(`Files to mutate: ${JSON.stringify(this.mutateFileNames, null, 2)}`);
+        log.debug(`Files to mutate: ${JSON.stringify(this.filesToMutate.map(_ => _.name), null, 2)}`);
```

Comment:
Funny, you use the temp variable `_` here while it's `file` in the line above

Reply:
Written the code on a different day of the week I guess. I'll align on `file`.